types/stripe-js/payment-intents.d.ts

```diff
@@ -96,6 +97,20 @@ export interface CreatePaymentMethodBancontactData
   };
 }
 
+export interface CreatePaymentMethodBillieData
+  extends PaymentMethodCreateParams {
+  /**
+   * Requires beta access:
+   * Contact [Stripe support](https://support.stripe.com/) for more information.
+   */
+  type: 'billie';
+
+  /**
+    * Details about the Billie pament method. Currently there are no supported child attributes for this field, but sending an empty object is mandatory.
+    */
+  billie: {}; // eslint-disable-line @typescript-eslint/ban-types
```

Comment:
If I don't pass in an empty object here, I get the following error back:

> Missing value for confirmBilliePayment: payment_method.billie should be an object or element.

Not sure if this is intentional, or if it will be fixed on Stripes side. E.g. `confirmKlarnaPayment` can be called without passing an empty `klarna: {}` object...